pkg/client/fake/client_test.go

```diff
@@ -2695,6 +2746,81 @@ var _ = Describe("Fake client", func() {
 	}
 })
 
+type WithSchemalessSpecList struct {
+	*metav1.ListMeta
+	*metav1.TypeMeta
+	Items []*WithPointerMeta
+}
+
+func (t *WithSchemalessSpecList) DeepCopy() *WithSchemalessSpecList {
+	l := &WithSchemalessSpecList{
+		ListMeta: t.ListMeta.DeepCopy(),
+	}
+	if t.TypeMeta != nil {
+		l.TypeMeta = &metav1.TypeMeta{
+
+			APIVersion: t.APIVersion,
+
+			Kind: t.Kind,
+		}
+	}
+	for _, item := range t.Items {
+		l.Items = append(l.Items, item.DeepCopy())
+	}
+
+	return l
+}
+
+func (t *WithSchemalessSpecList) DeepCopyObject() runtime.Object {
+	return t.DeepCopy()
+}
```

Comment:
Not actually used in the test, let's drop it?

(otherwise, it is using the wrong type "WithPointerMeta")